webapp/channels/src/components/shared_user_indicator.tsx

```diff
@@ -30,7 +30,7 @@ const SharedUserIndicator = (props: Props) => {
         <i
             data-testid='SharedUserIcon'
             className={classNames('icon icon-circle-multiple-outline', props.className)}
-            aria-label={props.ariaLabel || intl.formatMessage({id: 'shared_user_indicator.aria_label', defaultMessage: 'shared user indicator'})}
+            aria-label={props.ariaLabel || intl.formatMessage({id: 'shared_user_indicator.aria_label', defaultMessage: 'shared user'})}
```

Comment:
I removed the "indicator" from this because it's not really useful to the user, similar to how we were asked by LevelAccess to change labels like "online icon" to just "online"